templates/taxi/car_list_delete.html

```diff
@@ -0,0 +1,10 @@
+{% extends "base.html" %}
+
+{% block content %}
+  <h1>Delete Car</h1>
+  <p>Do you want to delete this {{ car.model }}?</p>
+  <form action="" method="post">
+    {% csrf_token %}
+    <input type="submit" value="Delete" class="btn btn-danger"/>
+  </form>
```

Comment:
Add cancel button which will lead to previous page

Reply:
not resolved